openapi.yaml

```diff
@@ -0,0 +1,734 @@
+      operationId: createDeveloper
+      tags:
+        - developers
+      parameters:
+        - in: body
+          name: body
+          required: true
+          schema:
+            $ref: '#/definitions/DeveloperPostBody'
+      responses:
+        '200':
+          description: Developer created successfully
+          schema:
+            $ref: '#/definitions/DeveloperResult'
+        '400':
+          description: Invalid developer object
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+  /developers/{developerId}:
+    parameters:
+      - $ref: '#/parameters/authorization'
+      - name: developerId
+        in: path
+        type: string
+        required: true
+    get:
+      summary: Get developer by its Id
+      operationId: getDeveloperById
+      tags:
+        - developers
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/DeveloperResult'
+        '404':
+          description: Developer not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    patch:
+      summary: Update developer by its Id
+      operationId: updateDeveloperById
+      tags:
+      - developers
+      parameters:
+        - name: body
+          in: body
+          schema:
+            $ref: '#/definitions/DeveloperPatchBody'
+      responses:
+        '200':
+          description: Developer updated successfully
+          schema:
+            $ref: '#/definitions/DeveloperResult'
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/definitions/Errors'
+        '404':
+          description: Developer not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    delete:
+      summary: Delete a developer record
+      operationId: deleteDeveloperById
+      tags:
+        - developers
+      responses:
+        '204':
+          description: Developer deleted successfully
+        '404':
+          description: Developer not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+  /reviews:
+    parameters:
+    - $ref: '#/parameters/authorization'
+    get:
+      summary: List all reviews
+      description: Get a list of all reviews or filter by query parameters
+      operationId: getReviews
+      tags:
+        - reviews
+      parameters:
+        - name: gameId
+          in: query
+          type: string
+          required: false
+          description: Filter by the Id of the game
```

Comment:
Are we only allow users to filter reviews by one `gameId` at a time? Can we search with all reviews for multiple games with one request? For example, how can we search for all review for game A and game B without making two requests to your API?

Reply:
changed to accept an array of strings